application/src/main/webapp/vue-app/messageFilter.js

```diff
@@ -11,11 +11,17 @@ export default function(msg, highlight, emojis) {
     words.forEach(w => {
       // check link
       if (w.indexOf('http:') === 0 || w.indexOf('https:') === 0 || w.indexOf('ftp:') === 0) {
```

Comment:
We need to refactor the function to improve security :

- Check if the contenty is a link by using the new URL function
```
try {let urlMessage = new URL(this.messageContent)} catch(urlError) {}
if(urlMessage) {
......
}
```

- External images should be managed the same way as external links